frontend/mainwindow.cpp

```diff
@@ -50,6 +50,11 @@ void MainWindow::initialiseElements() {
   viewMenu = ui->menubar->addMenu("View");
   ui->runDataTable->horizontalHeader()->setSectionsMovable(true);
   ui->runDataTable->horizontalHeader()->setDragEnabled(true);
+  ui->runDataTable->setAlternatingRowColors(true);
+  ui->runDataTable->setStyleSheet("alternate-background-color: #e7e7e6;");
+
+  connect(ui->groupButton, SIGNAL(toggled(bool)), this,
+          SLOT(groupButtonToggle(bool)));
```

Comment:
No need to connect this manually - you can still use the `clicked` slot for the button as the button check state is passed as a bool by that signal.